eip-0040.md

```diff
@@ -38,29 +38,30 @@ Anchoring `EIP-40-Asset-ID` to a specific ErgoTree means that we can have someth
 
 Querying proof of authorization can be done as follows `isAuthed(tokenId) := getBox(boxId=tokenId).isDefined && getUtxo(boxId=tokenId).isEmpty`
 
-In order to simplify the process of querying asset metadata we propose to store it in the PermittingBox. Then asset metadata can be queried as follows `metadata(EIP-40-Asset-ID(tokenId)) = parseEip40Metadata(getBox(boxId=tokenId))`, where `parseEip40Metadata` is a method that extracts all the EIP-4 fields from the given box, plus current amount of emission.
+In order to simplify the process of querying asset metadata we propose to store it in the PermittingBox. Then asset metadata can be queried as follows `metadata(EIP-40-Asset-ID(tokenId)) = parseEip40Metadata(getBox(boxId=tokenId))`, where `parseEip40Metadata` is a method that extracts all the EIP-4 fields from the given box, plus current circulating supply.
 
-Emission amount as well as other metadata members is written by permitting entity
+Circulating supply as well as other metadata members is written by permitting entity
 
 Let's define extended registers mapping from EIP-4:
 
-| Register        | Description                                     | Example                      |Encoded                      |
-| --------------- |:-----------------------------------------------:| ----------------------------:|----------------------------:|
-| R2              | Token id and amount pair                        | [("7d...09", 100000)]        |                             |
-| R4              | Token verbose name (UTF-8 representation)       | "USD"                        | "0e03555344"                |
-| R5              | Token description (UTF-8 representation)        | "Nothing backed USD token"   | "0e184e6f7468696e67206261636b65642055534420746f6b656e"  |
-| R6              | Number of decimals                              | "2"                          | "0e0132"                         |
-| R7              | Current emission                              | "1000000"                          | "0e0131303030303030"                         |
+| Register       | SType   | Description                                     | Example                     |
+|:--------------:|:-------:|:------------------------------------------------|:----------------------------|
+| R2             | SString | Token id and amount pair                        | [("7d...09", 100000)]       |
```

Comment:
SString is not supported type of register. It is only possible to store byte of UTF string as Coll[Byte].
I suggest to use ErgoScript type names to specify register types. Using SType from sigma is implementation dependent terminology. It will be confusing for Appkit user for example and in other SDKs other type names may be used.
ErgoScript type names will serve as a common ground.